docs/troubleshooting.md

```diff
@@ -0,0 +1,141 @@
+
+ ```bash
+ $ oc get pvc
+ NAME        STATUS    VOLUME   CAPACITY   ACCESS MODES   STORAGECLASS   AGE
+ lvms-test   Pending                                      lvms-vg1       11s
+ ```
+
+To troubleshoot the issue, inspect the events associated with the PVC. These events can provide valuable insights into any errors or issues encountered during the provisioning process.
+
+ ```bash
+ $ oc describe pvc lvms-test  
+ Events:
+   Type     Reason              Age               From                         Message
+   ----     ------              ----              ----                         -------
+   Warning  ProvisioningFailed  4s (x2 over 17s)  persistentvolume-controller  storageclass.storage.k8s.io "lvms-vg1" not found
+ ```
+
+### `LVMCluster` CR or the Logical Volume Manager Storage (LVMS) components are missing
+
+If you encounter a "storageclass.storage.k8s.io 'lvms-vg1' not found" error, verify the presence of the `LVMCluster` resource:
+ 
+ ```bash
+ $ oc get lvmcluster -n openshift-storage
+ NAME            AGE
+ my-lvmcluster   65m
+ ```
+
+If an LVMCluster resource is not found, you can create one based on your requirements. [Here](../config/samples/lvm_v1alpha1_lvmcluster.yaml) is a sample one you can use.
+
+```bash
+$ oc create -n openshift-storage -f https://github.com/openshift/lvm-operator/raw/main/config/samples/lvm_v1alpha1_lvmcluster.yaml
+```
+ 
+If an `LVMCluster` already exists, check if all the pods from LVMS are in `Running` state on `openshift-storage` namespace:
+ 
+ ```bash 
+ $ oc get pods -n openshift-storage
+ NAME                                  READY   STATUS    RESTARTS      AGE
+ lvms-operator-7b9fb858cb-6nsml        3/3     Running   0             70m
+ topolvm-controller-5dd9cf78b5-7wwr2   5/5     Running   0             66m
+ topolvm-node-dr26h                    4/4     Running   0             66m
+ vg-manager-r6zdv                      1/1     Running   0             66m
+ ```
+ 
+There should be one running instance of `lvms-operator` and `vg-manager`, and multiple instances of `topolvm-controller` and `topolvm-node` depending on number of nodes.
+
+#### `topolvm-node` is stuck in `Init:0/1`
+
+This indicates a failure in locating an available disk for LVMS utilization. To investigate further and obtain relevant information, review the logs of the `vg-manager` pod.
+
+```bash
+$ oc logs vg-manager -n openshift-storage
+```
+
+### Disk failure
+
+If you encounter a failure message such as "failed to check volume existence" while inspecting the events associated with the PVC, it indicates a potential issue related to the underlying volume or disk. This failure message suggests that there might be a problem with the availability or accessibility of the specified volume. Further investigation is recommended to identify the exact cause and resolve the underlying issue.
```

Comment:
```suggestion
If you encounter a failure message such as "failed to check volume existence" while inspecting the events associated with the `PVC` CR, it might indicate a potential issue related to the underlying volume or disk. This failure message suggests that there is problem with the availability or accessibility of the specified volume. Further investigation is recommended to identify the exact cause and resolve the underlying issue.
```